docs/docs/genai/eval-monitor/insights/analyze-experiment.mdx

```diff
@@ -0,0 +1,127 @@
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { MessageSquare, Target, BarChart3, ExternalLink, Bot, Users, CheckCircle, TrendingUp, Database, Book, Rocket, Scale } from "lucide-react";
+
+# Analyze Experiment
```

Comment:
@nsthorat Wondering if there's potentially a more eye-catching / attention-grabbing name to use in the docs sidebar. Something that makes it 100% obvious that this feature automatically identifies the most important issues and creates a prioritized plan for which ones to address next